candidatebot.py

```diff
@@ -92,7 +129,7 @@ def does_page_exist(page_to_query):
   except ValueError, ex:
     print "Couldn't parse JSON:", ex
 
-
+@rate_limited(EDIT_PAGES_PER_SECOND)
 def create_page(person, login_cookies):
   """Create a page if it doesn't exist. If it already exists, just silently
```

Comment:
Perhaps just outputting some kind of punctuation (like what we do for running tests in python) would be helpful? Just a suggestion, not necessary.